tools/gevent_linter.py

```diff
@@ -0,0 +1,41 @@
+__version__ = '0.0.1'
+
+
+class CodingChecker:
+    name = 'flake8_raiden_gevent'
+    version = __version__
+
+    def __init__(self, _tree, filename):
+        self.filename = filename
+
+    def read_file(self):
+        if self.filename in ('stdin', '-', None):
+            try:
+                # flake8 >= v3.0
+                from flake8.engine import pep8 as stdin_utils
+            except ImportError:
+                from flake8 import utils as stdin_utils
+            return stdin_utils.stdin_get_value().splitlines(True)
+
+        try:
+            import pycodestyle
+        except ImportError:
+            import pep8 as pycodestyle
+        return pycodestyle.readlines(self.filename)
+
+    def run(self):
+        try:
+            lines = self.read_file()
+
+            for lineno, line in enumerate(lines, start=1):
+                if 'gevent.spawn' in line and '=' not in line:
```

Comment:
We need to consider cases where the greenlet is `append`ed to some list, and `Greenlet` instantiation + `start` case too, but this is a good one!